ATTRIBUTION.md

```diff
@@ -0,0 +1,21 @@
+# Attribution

```

Comment:
Attribution or Attributions?

Reply:
Gdude said Attribution. I has Attributions for a while. Not sure if it matters a whole bunch, but matching the file name is probably better